### PR TITLE
Make free coaching call url configurable and popup translatable

### PR DESCRIPTION
**Pull request by CamilleLegeron**
# Context

One user of our ANCT instance used the coaching call url that give a calendly link of gristlabs. So, we wanted to make it configurable by instance with the gristlabs calendly by default.

### CODE REVIEW

File: app/client/ui/WelcomeCoachingCall.ts [hunk 1]
```diff
@@ -77,35 +78,39 @@ export function showWelcomeCoachingCall(triggerElement: Element, appModel: AppMo
...
         bigPrimaryButtonLink(
-          'Schedule Call',
+          t('Schedule Call'),
           dom.on('click', () => {
             dismissPopup(false);
             logTelemetryEvent('clickedScheduleCoachingCall');
           }),
           {
-            href: FREE_COACHING_CALL_URL,
+            href: getFreeCoachingCallUrl(),
```
Comment (georgegevoian) [inline app/client/ui/WelcomeCoachingCall.ts:107]:
```suggestion
            href: getGristConfig().freeCoachingCallUrl,
```

This way, you don't need `getFreeCoachingCallUrl` in `gristUrls.ts`.

File: app/client/ui/WelcomeCoachingCall.ts [hunk 2]
```diff
@@ -77,35 +78,39 @@ export function showWelcomeCoachingCall(triggerElement: Element, appModel: AppMo
...
+      cssPopupTitle(t('free coaching call'),
+        dom.style('text-transform', 'capitalize'),
+        testId('popup-title')
+      ),
       cssPopupBody(
         cssBody(
           dom('div',
-            'Schedule your ', cssBoldText('free coaching call'), ' with a member of our team.'
+            t('Schedule your {{freeCoachingCall}} with a member of our team.',
+              {freeCoachingCall: cssBoldText('free coaching call')}
```
Comment (georgegevoian) [inline app/client/ui/WelcomeCoachingCall.ts:89]:
```suggestion
              {freeCoachingCall: cssBoldText(t('free coaching call'))}
```

File: app/server/lib/sendAppPage.ts [hunk 3]
```diff
@@ -54,6 +54,7 @@ export function makeGristConfig(options: MakeGristConfigOptions): GristLoadConfi
     baseDomain,
     singleOrg: process.env.GRIST_SINGLE_ORG,
     helpCenterUrl: process.env.GRIST_HELP_CENTER || "https://support.getgrist.com",
+    freeCoachingCallUrl: process.env.FREE_COACHING_CALL_URL || "https://calendly.com/grist-team/grist-free-coaching-call",
```
Comment (georgegevoian) [inline app/server/lib/sendAppPage.ts:57]: Could you add `FREE_COACHING_CALL_URL` to the list in the README? Thanks.

**Comment by CamilleLegeron** [comment]
Thanks @georgegevoian for the review. I made the modifications

File: README.md [hunk 4]
```diff
@@ -263,6 +263,7 @@ GRIST_FORCE_LOGIN    | Much like GRIST_ANON_PLAYGROUND but don't support anonymo
 GRIST_SINGLE_ORG | set to an org "domain" to pin client to that org
 GRIST_TEMPLATE_ORG | set to an org "domain" to show public docs from that org
 GRIST_HELP_CENTER | set the help center link ref
+FREE_COACHING_CALL_URL | set the link to the human help (exemple: email or meeting scheduling tool)
```
Comment (georgegevoian) [inline README.md:266]:
```suggestion
FREE_COACHING_CALL_URL | set the link to the human help (example: email or meeting scheduling tool)
```

**Review by georgegevoian — COMMENTED** [review]
Thanks @CamilleLegeron! Just one more small comment.

**Pull request merged**